test/expect/TestSparseCSRCPU.test_sparse_csr_print_cpu.expect

```diff
@@ -72,22 +72,25 @@ tensor([-0.2544, -0.2462, -0.9784,  0.8910,  0.5322, -0.4732, -0.6239,  0.0348,
 # values_shape: torch.Size([10])
```

Comment:
`crow_indices` are all zero from this point forward in the original expect file. not sure if there's a copy/paste issue. plz review.